ontology/cheminf.md

```diff
@@ -25,5 +25,18 @@ publications:
   title: 'The chemical information ontology: provenance and disambiguation for chemical data on the biological semantic web'
 repository: https://github.com/semanticchemistry/semanticchemistry
 tracker: https://github.com/semanticchemistry/semanticchemistry/issues
+usages:
+- description: ChEMBL uses CHEMINF in the RDF download
+  examples:
+  - description: Physicochemical properties are represented as classes that are typed with CHEMINF classes
+    url: http://rdf.ebi.ac.uk/resource/chembl/molecule/CHEMBL6939#mw_freebase
```

Comment:
This does not resolve for me? Certificate problems?

Reply:
ping @BZdrazil